VogtPayroll3/Employee.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+
+namespace VogtPayroll3
+{
+    class Employee
+    {
+        // Fields
+        private string _firstName;
+        private string _lastName;
+        private int _empID;
+        private int _hoursWorked;
+        private decimal _payrate;
+        private const decimal taxAmount = .30m;
+
+        public string FirstName { get => _firstName; set => _firstName = value; }
+        public string LastName { get => _lastName; set => _lastName = value; }
+        public int EmpID { get => _empID; set => _empID = value; }
+        public int HoursWorked { get => _hoursWorked; set => _hoursWorked = value; }
+        public decimal Payrate { get => _payrate; set => _payrate = value; }
+
+        // CTOR
+        public Employee(string firstName, string lastName, int empID, int hoursWorked, decimal payRate)
+        {
+            this.FirstName = firstName;
```

Comment:
No reason to do `this.` unless they have the same exact variable name. This is just a minor styling nitpick.